octodns/provider/dnsimple.py

```diff
@@ -290,6 +306,27 @@ def populate(self, zone, target=False, lenient=False):
                       len(zone.records) - before, exists)
         return exists
 
+    def supports(self, record):
+        # DNSimple does not support empty/NULL SRV records
+        #
+        # Fails silently and leaves a corrupt record
+        #
+        # Skip the record and continue
+        if record._type == "SRV":
+            if 'value' in record.data:
+                targets = (record.data['value']['target'],)
+            else:
+                targets = [value['target'] for value in record.data['values']]
+
+            if "." in targets:
+                self.log.warning(
+                    'supports: unsupported %s record with target (%s)',
+                    record._type, targets
+                )
+                return False
+
+        return record._type in self.SUPPORTS
```

Comment:
It's preferable to go ahead and call up to the super method here rather than duplicate it's check in case that ever changes and gets more involved in the future. 

```suggestion
        return super(DnsimpleProvider, self).supports(record)
```